src/writer/statement.ts

```diff
@@ -29,9 +29,7 @@ export async function action(body, ipfs, receipt, id): Promise<void> {
     created
   };
 
-  const query =
-    'INSERT IGNORE INTO statements SET ? ON DUPLICATE KEY UPDATE ipfs = ?, about = ?, statement = ?, updated = ?';
-  const params = [item, item.ipfs, item.about, item.statement, item.created];
+  const query = 'INSERT INTO statements SET ?';
```

Comment:
This should update existing statement (in case of duplicate key)

Reply:
In what scenario there can be duplicate key ?

In the prevent action replay PR, we ensured that the ID does not exist

Reply:
Yea yea this for `statements` table here primary key is `PRIMARY KEY (delegate, space),` if we receive a statement from same delegate we update existing record

Reply:
Mhhh. did not see those `PRIMARY KEY`. PR need more works to take these into account.